src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
```

Comment:
Давай переместим логику определения случайного индекса в метод getRandomColor(), мне кажется это не то значение, которое должно принадлежать экземпляру класса ColorSupplier. А вот переменную Random мы можем ынести на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод getRandomColor()

```suggestion
    Random random = new Random();
```